### ISSUE TITLE
D0 and D1 cannot be used as output pins on FRDM_K64F

### ISSUE THREAD

**Issue by xiaolongx** [01org/zephyr.js#514]
Flashing [test-k64f-pins.js](https://github.com/01org/zephyr.js/blob/master/tests/test-k64f-pins.js) on FRDM_K64F with commit a162db1, D0 and D1 cannot be used as output pins.

Output result:
```
Test K64f_pins APIs
PASS - K64f_pins: D0 defined
PASS - K64f_pins: D0 input
FAIL - K64f_pins: D0 output
PASS - K64f_pins: D1 defined
PASS - K64f_pins: D1 input
FAIL - K64f_pins: D1 output
```
Branch | Build ID | Date | Result
---|---|---|---
master | 5844f9f | Dec 7, 2016 | PASS
master | a162db1  | Dec 7, 2016 | FAIL

**Comment by qiaojingx** [comment]
Another issue #665  is reproduced on FRDM-K64F with commit 3f8eb85.
Will verify this issue after #665 fixed.

**Comment by grgustaf** [comment]
I can reproduce that D0 and D1 don't work as inputs, but e.g. D3 does.

I also so far can't seem to use any of the pins as an output. I haven't tried this for a while. I guess I'll look into the two commits you list.

**Comment by grgustaf** [comment]
Fixed by #954.

**Comment by qiaojingx** [comment]
Verified with commit f3d587d + #954. This issue is not reproducible.
Will close it after patch merged.
